src/controllers/dynakube/version/reconciler.go

```diff
@@ -72,6 +74,37 @@ func (reconciler *Reconciler) updateVersionStatuses(ctx context.Context, updater
 			return err
 		}
 	}
+
+	err = SetOneAgentHealthcheck(ctx, reconciler.apiReader, registry.NewClient(), reconciler.dynakube, reconciler.dynakube.OneAgentImage(), dockerConfig.RegistryAuthPath)
```

Comment:
Not 100% happy of the location here, but I had to do it because of the dockerConfig cleanup on this function (line 66)

Reply:
Fixed!